### PR TITLE
Implementation of RBAC Overview page

### PR DESCRIPTION
**Pull request by radekkaluzik**
https://issues.redhat.com/browse/RHCLOUD-30302

### CODE REVIEW

File: src/Routing.tsx [hunk 1]
```diff
@@ -36,6 +38,10 @@ const RemoveServiceAccountFromGroup = lazy(() => import('./smart-components/grou
 const QuickstartsTest = lazy(() => import('./smart-components/quickstarts/quickstarts-test'));
 
 const getRoutes = ({ enableServiceAccounts }: Record<string, boolean>) => [
+  {
```
Comment (karelhala) [inline src/Routing.tsx:41]: Is the nav hidden behind feature flag?

**Comment by radekkaluzik** [comment]
Screenshot 2024-01-24 at 13 22 27
Screenshot 2024-01-24 at 13 22 04

**Review by karelhala — APPROVED** [review]
Looking really good!

File: src/Routing.tsx (hunk 1 above)
Comment (radekkaluzik) [reply src/Routing.tsx:41]: Yes, it is.

**Review by fhlavac — APPROVED** [review]
Looks good!

**Pull request merged**